index.js

```diff
@@ -33,7 +33,7 @@ app.get('/', (req, res) => res.status(200).send({
 app.use('/api/v1/auth', auth);
 app.use('/api/v1/auth_twitter', twitterRouter);
 app.use('/api/v1/profiles', profiles);
-app.use('/api/v1/forgotpassword', user);
+app.use('/api/v1/passwords', passwords);
```

Comment:
what is `password`? You can use a more descriptive label

Reply:
can i use forgot password instead? @Chidi3b3r3

Reply:
basically i used password because of the various password methods i have in the route..because i have both 'reset password method' and 'forgot password method'

Reply:
Understood

Reply:
Understood